internal/pkg/cli/task_run.go

```diff
@@ -681,6 +681,9 @@ Did you tag your secrets with the "copilot-application" and "copilot-environment
 		if err := o.displayLogStream(); err != nil {
 			return err
 		}
+		if nonZeroExitCodeErr := o.runner.CheckNonZeroExitCode(tasks); nonZeroExitCodeErr != nil {
```

Comment:
```suggestion
		if err := o.runner.CheckNonZeroExitCode(tasks); err != nil {
```
it is ok to just use `err` here because its scope is very small (only for this condition)